main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -258,9 +272,9 @@ object Main {
   /**
     * Extracts the exit code from the given result.
     */
-  private def getCode(result: Result[_, Int]): Int = result match {
+  private def getCode[T, E](result: Result[T, E]): Int = result match {
```

Comment:
Maybe this function should be removed. I'll have to double check if it's still used somewhere

Reply:
Yes, lets remove it.